soot-infoflow-android/src/soot/jimple/infoflow/android/entryPointCreators/AndroidEntryPointCreator.java

```diff
@@ -326,6 +338,60 @@ protected SootMethod createDummyMainInternal() {
 		return mainMethod;
 	}
 
+	private void createJavascriptCallbacks() {
+		Jimple j = Jimple.v();
+		Hierarchy h = Scene.v().getActiveHierarchy();
+		for (SootMethod m : javascriptInterfaceStmts.keySet()) {
+			Set<Stmt> statements = javascriptInterfaceStmts.get(m);
+			for (Stmt s : statements) {
+				UnitPatchingChain units = m.retrieveActiveBody().getUnits();
+				Stmt succ = (Stmt) units.getSuccOf(s);
```

Comment:
Wenn wir irgendwann mehr instrumentieren, geht das zu Bruch. Ich würde mir eher in einem Tag am aktuellen Statement den Feldnamen merken,